components/x-styling-demo/src/Button.jsx

```diff
@@ -0,0 +1,13 @@
+import { h } from '@financial-times/x-engine';
+import buttonStyles from './Button.css';
+import classNames from 'classnames';
+
+export const Button = ({large, danger}) => <button
+	className={classNames(
+		buttonStyles.button,
+		{
+			[buttonStyles.large]: large,
+			[buttonStyles.danger]: danger,
+		}
+	)}
```

Comment:
What's the benefit of using the `classnames` module in this context? Is that something we want to promote?

Reply:
(this could make it appear as though it is mandatory to use the module in order to add styles)

Reply:
so i like it because it turns what is usually a bunch of string concatenation into a thing that looks like a thing. it's by no means mandatory, but i would like to see people using it for complex cases like this